<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContractorWorkerDataClass</class>
 <widget class="QWidget" name="ContractorWorkerDataClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1272</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ContractorWorkerData</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="p_worker">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLineEdit" name="le_quickSearch"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lb_quickSearch">
           <property name="text">
            <string>Quick Search</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pb_new">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="pb_delete">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tv_worker"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="p_workerEdit">
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Company Data</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="lb_companyName">
                <property name="text">
                 <string>Company Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_companyName"/>
              </item>
              <item>
               <widget class="QPushButton" name="pb_selectCompany">
                <property name="text">
                 <string>Select Company</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Worker Data</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="4" column="0">
                 <widget class="QLabel" name="lb_note">
                  <property name="text">
                   <string>Note</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="le_firstName"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLineEdit" name="le_lastName"/>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="le_email"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="lb_lastName">
                  <property name="text">
                   <string>Last Name</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="lb_phone">
                  <property name="text">
                   <string>Phone</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lb_firstName">
                  <property name="text">
                   <string>First Name</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="le_phone"/>
                </item>
                <item row="5" column="0" colspan="2">
                 <widget class="QTextEdit" name="te_note"/>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lb_email">
                  <property name="text">
                   <string>E-Mail</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="pb_save">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pb_back">
               <property name="text">
                <string>Back</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
